### PR TITLE
feat: add wrong network

### PR DESCRIPTION
**Pull request by paulobarcelosbr**
image

### CODE REVIEW

File: src/locales/en/common.json [hunk 1]
```diff
@@ -29,6 +29,8 @@
   "apr": "APR",
   "pnl": "PNL",
   "rewardsFee": "Rewards Fee",
+  "wrongNetwork": "Wrong Network",
+  "switch": "Switch",
```
Comment (arthurmr96) [inline src/locales/en/common.json:33]:
```suggestion
  "switchTo": "Switch to",
```

File: src/locales/pt/common.json [hunk 2]
```diff
@@ -29,6 +29,8 @@
   "apr": "APR",
   "pnl": "PNL",
   "rewardsFee": "Taxa de Recompensas",
+  "wrongNetwork": "Rede Errada",
+  "switch": "Trocar",
```
Comment (arthurmr96) [inline src/locales/pt/common.json:33]:
```suggestion
  "switchTo": "Trocar para",
```

**Review by arthurmr96 — CHANGES_REQUESTED** [review]

File: src/components/stake/StakeForm.tsx [hunk 3]
```diff
@@ -58,27 +59,55 @@ export function StakeForm({ type, accountAddress, poolAddress }: StakeFormProps)
...
+        disabled={isNetworkWrong || isLoading}
         purple={type === 'withdraw'}
         hasError={insufficientFunds}
       />
       <StakeButton
         isLoading={isLoading}
-        onClick={action}
-        label={insufficientFunds ? errorLabel : actionLabel}
+        onClick={handleActionButton}
+        label={handleLabelButton()}
```
Comment (0xfabricio) [inline src/components/stake/StakeForm.tsx:110]: handleLabelButton

Comment (0xfabricio) [reply src/components/stake/StakeForm.tsx:110]: remover ()

File: src/config/chain.ts [hunk 4]
```diff
@@ -32,7 +32,7 @@ const configs: ChainConfig[] = [
   // },
   {
     chainId: 5,
-    name: 'goerli',
+    name: 'Goerli',
```
Comment (0xfabricio) [inline src/config/chain.ts:35]: goerli, transforma capitalize no componente

**Review by 0xfabricio — CHANGES_REQUESTED** [review]
Faz as atualizações e me avisa

**Review by arthurmr96 — APPROVED** [review]

**Review by 0xfabricio — APPROVED** [review]

**Comment by 0xfabricio** [comment]
Screenshot 2023-06-13 at 1 49 49 PM

If the wallet is disconnect, 
should appear connect wallet
switch goerli
select community

**Pull request merged**